projects/ion/src/lib/input/input.component.html

```diff
@@ -75,3 +75,7 @@
     <ion-icon type="close-solid"></ion-icon>
   </button>
 </div>
+
+<div class="errorMsg" *ngIf="invalid">
```

Comment:
```suggestion
<div class="errorMsg" *ngIf="invalid && errorMsg">
```

Reply:
Isso evita adicionar uma div com um p que não existe quando for invalido e não tiver uma msg de erro.